src/components/Tabs/Tab.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.css';
+
+function Tab(props) {
```

Comment:
Maybe just construction with arrow function? It would be more consistent with other stateless components.